app.organicmaps.desktop.yml

```diff
@@ -11,6 +11,10 @@ finish-args:
   - --socket=fallback-x11
   - --socket=wayland
   - --device=dri
+cleanup:
+  # rsync was only needed for building
+  - '/bin/rsync'
+  - '/bin/rsync-ssl'
 modules:
   - name: rsync # used at build time
```

Comment:
What is it used for exactly?

Reply:
That is an absolutely good point!
At first glance, it appears that it is only used for [some testing here](https://github.com/organicmaps/organicmaps/blob/master/3party/opening_hours/opening_hours_integration_tests/CMakeLists.txt#L10), that IMHO we shouldn't even build.
So perhaps we should not merge this, but configure the build to avoid tests, and get rid of the `rsync` module entirely.

Reply:
1. `rsync -a` can be replaced with `cp -a` to remove the unnecessary dependency.
2. It would be great if you can run as many unit tests (in release) as possible to actually prove that the release doesn't have any issues on that architecture/platform/environment.

Reply:
As far as I remember, rsync was used to avoid unnecessary copying of the file. Also, these integration tests are failing now. So you shouldn't run them yet.

Reply:
All right. So for `1.` there is https://github.com/organicmaps/organicmaps/pull/2730 ready for review.